### PR TITLE
[YUNIKORN-2609] Improve visual style of the Web UI

### PR DESCRIPTION
**Pull request by SP12893678**
### What is this PR for?
Improve visual style of the Web UI
The full design can be previewed on this link: [ [DESIGN](https://xd.adobe.com/view/1d84899f-72a8-472f-b03f-de40451b0956-48d7/) ]

- Fix padding/margin values
- Add rounding on elements to match the design (menu selection, dropdowns, etc)
- Fix font weight on visual elements to match the design

Detail:
- Sidebar navigation UI
- Header right top help button change icon
- Global => Form Field (Input, Select) to be density
- Dashboard page
   - Chart Card title & content => padding
   - Chart Card title => font size & font weight
   - Application Status & Container Status chart-legend => font weight
- Applications page
   - Table header row =>  font weight
- Nodes page
   - Table header row =>  font weight
   - indicator field => fix align & remove extra gap

### What type of PR is it?
* [x] - Improvement

### What is the Jira issue?
[YUNIKORN-2609](https://issues.apache.org/jira/browse/YUNIKORN-2609)

### Screenshots (if appropriate)
| Before | After | 
|  ----  | ----  |
| <video src="https://github.com/user-attachments/assets/6e2c1a4b-d75f-41b9-9783-dea8351b73e3"> | <video src="https://github.com/user-attachments/assets/d4d8cfeb-daa8-4757-a696-eba594e0360c"> |

### CODE REVIEW

File: src/app/components/nodes-view/nodes-view.component.html
Comment (chenyulin0719) [inline src/app/components/nodes-view/nodes-view.component.html:37]: The example "(Ex. amd64)" is meaningless to me. Do we need it?

image

**Review by chenyulin0719 — COMMENTED** [review]
Thanks for the patch.  Overall LGTM. Only one minor question.

File: src/app/components/nodes-view/nodes-view.component.html
Comment (chenyulin0719) [reply src/app/components/nodes-view/nodes-view.component.html:37]: I thought the "amd64" doesn't provide any useful information here.

**Review by chenyulin0719 — APPROVED** [review]
+1 LGTM

**Comment by chenyulin0719** [comment]
Hold this PR for a few days to see if @dcoric or @wilfred-s  have capacity to double confirm.

**Comment by craigcondit** [comment]
> +1 LGTM I have tested it locally and it looks good.

Only committers should be adding +1/-1 to comments.

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**
